Doc/library/collections.abc.rst

```diff
@@ -280,6 +287,11 @@ Collections Abstract Base Classes -- Detailed Descriptions
 
    ABCs for read-only and mutable :term:`mappings <mapping>`.
 
+   Example use in type annotations::
+
+      def get_position_in_index(word_list: Mapping[str, int], word: str) -> int:
+          return word_list[word]
+
```

Comment:
Maybe we should just get rid of this... it doesn't belong in the stub entry for the deprecated alias in `typing`, but it feels out of place here since these docs don't otherwise talk about type annotations at all. And it's too small to deserve its own section in the typing docs, like we're doing for generators/coroutines.

We could also maybe get rid of the "This type may be used as follows:" sections from the stub entries for `typing.List` and `typing.Dict`, as it sort-of implies that it's recommended to use that type, which isn't true. (But for both of those types, I'd keep the "Note that to annotate arguments" warning paragraph.)

Reply:
And this is a good one, `List` and `Dict` shouldn't suggest such usage.